SAN/code/network_simulations/quadrature_timing.jl

```diff
@@ -0,0 +1,199 @@
+using Pkg
+Pkg.add("Quadrature")
```

Comment:
- [ ] Add unit tests with `@test` for all the cases in which it is possible to compute the answer by hand. Or perhaps write a separate test file (so that the automated runs do not error out?)
- [ ] Revamp to use Pkg.activate. When using Pkg.activate, we should commit the toml files. For more, see comments for `benchmark_timing.jl` in [NC-64 pull request](https://github.com/fernando-duarte/networks_hub/pull/10) regarding package environments.
- [ ] Commit bash scripts that do the automated runs (I think you have them? Since I do see some quadrature output files being generated in the EC2 instance?)
- [ ] Double check units. See [my comment in NC-64 pull request review](https://github.com/fernando-duarte/networks_hub/pull/10#discussion_r603720344)

Reply:
I put tests in not using the @test to avoid the automated run erroring. 
I updated the script twith the new Pkg.activate syntax. 
I commited the bash scripts. Sorry for forgetting to commit those last run. 
I checked units - but I agree with the max memory concern.